scene/main/viewport.cpp

```diff
@@ -2789,7 +2789,11 @@ bool Viewport::_sub_windows_forward_input(const Ref<InputEvent> &p_event) {
 
 	gui.subwindow_focused->_window_input(ev);
 
-	return true;
+	if (gui.subwindow_focused) {
```

Comment:
This is always true, no?

It's checked above already, and the previous line dereferences it. Unless `gui.subwindow_focused->_window_input(ev);` can turn it to `nullptr`.